app/sidekiq/quote_job.rb

```diff
@@ -0,0 +1,12 @@
+class QuoteJob
+  include Sidekiq::Job
+  sidekiq_options retry: 5
+
+  def perform(ticker, source_index = rand(3))
+    if asset_value = GetAssetValueService.new(ticker, source_index).value
+      Quote.create!(asset: Asset.find_by(symbol: ticker), price: asset_value)
```

Comment:
não da pra reencrever? `Asset.find_by(symbol: ticker).quotes.create!(price: asset_value)`

Reply:
Asset didn't have has_many quotes, but it was added